### ISSUE TITLE
Animation Curves Timeline Not Supported

### ISSUE THREAD

**Issue by winderine** [Falano/kritaSpritesheetManager#28]
I've only been using this script for a few days, and at first it worked brilliantly for exporting pixel art as minecraft spritesheets. It automatically detected how many frames I had set up and exported without any issues. 

Yesterday or the day before, it stopped detecting the length of my animations. I started having to manually input the last frame. Well, it took me a minute to figure that out, but still useful. I didn't know why, though. 

Today, however, I couldn't get it to work at all. It continually exported the last frame as the only sprite. Now, this animation is a real simple sparkle where I have one layer fading in and out over the other to interpolate a sort of magical twinkle, ... and as I was typing this, I thought, "so maybe it's not working because the keyframes are only dots on the timeline, not gray squares? Maybe it's not recognizing animation curve keyframes at all?"

To test this I created a new layer with a single pixel on it, then created "duplicate keyframes" at the start and end of my Animation Timeline docker. This time running the script worked, I have a spritesheet that reflects what I see when clicking the play button!

TLDR: script requires keyframes to be in the animation timeline and doesn't register keyframes in the animation curves timeline.
